packages/core/components/AnnotationFilterForm/index.tsx

```diff
@@ -30,11 +41,32 @@ interface AnnotationFilterFormProps {
 export default function AnnotationFilterForm(props: AnnotationFilterFormProps) {
     const dispatch = useDispatch();
     const allFilters = useSelector(selection.selectors.getFileFilters);
+    const fuzzyFilters = useSelector(selection.selectors.getFuzzyFilters);
+    const anyValueFilters = useSelector(selection.selectors.getIncludeFilters);
+    const noValueFilters = useSelector(selection.selectors.getExcludeFilters);
     const annotationService = useSelector(interaction.selectors.getAnnotationService);
     const [annotationValues, isLoading, errorMessage] = useAnnotationValues(
         props.annotation.name,
         annotationService
     );
+    const usingAnyValueFilter: boolean = React.useMemo(() => {
```

Comment:
This logic is reused a lot could do something like 
```
const isUsingFilter = (filters: Filter[], annotationName: string) => {
    return filters?.some((filter) => filter.annotationName === annotationName) || false;
};
```

Reply:
The "anyValue" naming is a little misleading here, we're actually checking if we're using an `INCLUDE` (any value) filter rather than a `Filter` of any value